buf/registry/module/v1beta1/upload_service.proto

```diff
@@ -40,9 +40,12 @@ message UploadRequest {
     ModuleRef module_ref = 1 [(buf.validate.field).required = true];
     // The commit_id of the Commit, if this is referencing a Commit that already exists.
     //
-    // If the ModuleRef refers to a Module that has associated Content, this field should *not*
-    // be set, and setting it is an error.
-    string commit_id = 2 [(buf.validate.field).string.uuid = true];
+    // If the ModuleRef refers to a Module that has associated Content, this field should be empty,
+    // and setting a value is an error.
+    string commit_id = 2 [
+      (buf.validate.field).string.uuid = true,
+      (buf.validate.field).ignore_empty = true
```

Comment:
Ah I see, I didn't realize empty string (i.e., unset) did not pass validation. https://github.com/bufbuild/protovalidate/blob/8074ae2f42417345ef103d83fb62e4245010715d/proto/protovalidate/buf/validate/validate.proto#L2898-L2902
I'm not sure what's the way we want to go here, so I'll defer to Peter.

Reply:
For reference:
- some URLs use this model: https://github.com/bufbuild/registry-proto/blob/97279adc5b464d8296b2731ad22ddb2f2cfc253c/buf/registry/legacy/federation/v1beta1/upload_service.proto#L93-L101
- some URLs have both presence and `ignore_empty`: https://github.com/bufbuild/registry-proto/blob/97279adc5b464d8296b2731ad22ddb2f2cfc253c/buf/registry/module/v1beta1/module_service.proto#L165-L170
- some descriptions use just optional: https://github.com/bufbuild/registry-proto/blob/97279adc5b464d8296b2731ad22ddb2f2cfc253c/buf/registry/owner/v1beta1/organization_service.proto#L149-L150

It'd be nice to get them all to a consistent state. I think presence should be the way to go, unless the zero value is actually valid **and different** than unset, but for URLs and description that doesn't seem like the case. Wdyt @bufdev